client.go

```diff
@@ -271,3 +279,98 @@ func (c *SingleServerClient) Shutdown() {
 	var reply ShutdownResponse
 	c.client.Call("RpcServer.Shutdown", &args, &reply)
 }
+
+// MultiServerClient
+
+func (c *MultiServerClient) Build() error {
+	return build(c.app.BuildCmd(), c.appDir)
+}
+
+func (c *MultiServerClient) Push(deployId string) error {
+	for _, c := range c.clients {
+		if err := c.Push(deployId); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *MultiServerClient) Run(deployId string) error {
+	for _, c := range c.clients {
+		if err := c.Run(deployId); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *MultiServerClient) Stop(deployId string) error {
+	for _, c := range c.clients {
+		if err := c.Stop(deployId); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *MultiServerClient) SetMainByPort(port int) error {
+	// Only makes sense if you are connecting to a single backend
+	// server
+	if len(c.clients) > 1 {
+		return fmt.Errorf("Cannot set current app by port when target is a group of machines")
+	}
+
+	return c.clients[0].SetMainByPort(port)
+}
+
+func (c *MultiServerClient) SetMainById(deployId string) error {
+	for _, c := range c.clients {
+		if err := c.SetMainById(deployId); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *MultiServerClient) ListDeploys() ([]*Deploy, error) {
+	var deploys []*Deploy
+
+	for _, c := range c.clients {
+		if deploysForServer, err := c.ListDeploys(); err != nil {
+			return nil, err
+		} else {
+			deploys = append(deploys, deploysForServer...)
+		}
+	}
+
+	return deploys, nil
+}
+
+func (c *MultiServerClient) KillUnknownProcesses() {
```

Comment:
Shutdown is used by the integration test code (so we need it as part of the API), however, it should be removed from the terminal client: https://github.com/helix-collective/camus/issues/49

Agree that killUnknownProcesses is dangerous and should be removed altogether. I've raised a ticket for this: https://github.com/helix-collective/camus/issues/48